examples/neural_style_transfer.py

```diff
@@ -72,13 +72,13 @@
 content_weight = 0.025
 
 # dimensions of the generated picture.
-img_width = 400
-img_height = 400
-assert img_height == img_width, 'Due to the use of the Gram matrix, width and height must match.'
+img_nrows = 400
+img_ncols = 400
```

Comment:
The use of  `width` and `height` is more readable than `img_nrows` / `img_ncols`, is it not?